src/chart/layout/addLegend.js

```diff
@@ -0,0 +1,10 @@
+export default function addLegend() {
+    if (!this.parent)
```

Comment:
Might be worth a comment here explaining why the `if(!this.parent)` is needed.

Reply:
Sure thing.